### PR TITLE
All tests now pass! We now support PGP in Larky! 🍾🎆

### PR DESCRIPTION
**Pull request by mahmoudimus**
1. Cleanup by removing __init__<a href="2.star">2.star</a>
2. Switch from '%02X' format string to using `hexlify`

---

**Stack**:
- #287 ⮜
- #286
- #285
- #284
- #283
- #282
- #281
- #280
- #279
- #278
- #277
- #276
- #274
- #273
- #272
- #271
- #270
- #269
- #268
- #267
- #266
- #265
- #264
- #263
- #262
- #261
- #260
- #259

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: larky/src/main/resources/vendor/OpenPGP/__init__.star [hunk 1]
```diff
@@ -605,72 +624,52 @@ def _class_Message():
...
+                    # print("in __iter__, parsed packet:", packet)
+                    # print("in __iter__, input has more?", self._input.has_more())
                     if packet:
                         self._packets_start.append(packet)
-                        self.idx += 1
                         return Ok(packet)
                     else:
                         fail("OpenPGPException: Parsing is stuck")
-            print("where did i get here?", self.idx)
+            # print("where did i get here?", self.idx)
```
Comment (aslepakurov) [inline larky/src/main/resources/vendor/OpenPGP/__init__.star:655]: We need to remove all the prints/comment-out prints

**Review by aslepakurov — APPROVED** [review]

File: larky/src/main/resources/vendor/OpenPGP/__init__.star (hunk 1 above)
Comment (mahmoudimus) [reply larky/src/main/resources/vendor/OpenPGP/__init__.star:655]: I can do that now. Would you want that now?

File: larky/src/main/resources/vendor/OpenPGP/__init__.star [hunk 2]
```diff
@@ -983,6 +976,199 @@ def _class_AsymmetricSessionKeyPacket():
...
+    body += pack("!B", node.hash_algorithm)
+    body += pack("!H", node.hash_head)
+
+    for mpi in node.data:
+        body += pack("!H", bitlength(mpi)) + mpi
+
+    return body
+
+
+def to_bytes3(p, trailer):
```
[the remark this answers is not in the document]
Comment (mahmoudimus) [reply larky/src/main/resources/vendor/OpenPGP/__init__.star:1021]: x

File: larky/src/main/resources/vendor/OpenPGP/__init__.star (hunk 1 above)
Comment (aslepakurov) [reply larky/src/main/resources/vendor/OpenPGP/__init__.star:655]: That's ok. We need a massive refactoring/cleaning up

**Pull request merged**